test/wasm2c/tail-calls.txt

```diff
@@ -0,0 +1,858 @@
+;;; TOOL: run-wasm2c
+;;; ARGS0: --debug-names
+;;; ARGS*: --enable-tail-call
+(type $i32_f32 (func (param i32 f32)))
```

Comment:
Any chance we can use a test with a return value as well. It would be nice to see the codegen for this part.

Reply:
Sure, let me work on this.

Reply:
Just pushed -- agreed that this is illustrative (it also caught a missing newline).